docs/api/non_geo_datasets.csv

```diff
@@ -24,6 +24,7 @@ Dataset,Task,Source,License,# Samples,# Classes,Size (px),Resolution (m),Bands
 `LoveDA`_,S,Google Earth,"CC-BY-NC-SA-4.0","5,987",7,"1,024x1,024",0.3,RGB
 `MapInWild`_,S,"Sentinel-1/2, ESA WorldCover, NOAA VIIRS DNB","CC-BY-4.0",1018,1,1920x1920,10--463.83,"SAR, MSI, 2020_Map, avg_rad"
 `Million-AID`_,C,Google Earth,-,1M,51--73,,0.5--153,RGB
+`MMEarth`_,"C, S", Google Earth,"CC-BY-4.0","100K - 1M",,"128x128 or 64x64","multiple modalities",MSI
```

Comment:
```suggestion
`MMEarth`_,"C, S", Google Earth,"CC-BY-4.0","100K--1M",,"128x128 or 64x64","multiple modalities",MSI
```
This will create an en-dash for ranges.